src/components/pathway-progress/SubscriptionStatusCard.jsx

```diff
@@ -28,12 +29,40 @@ const SubscriptionStatusCard = () => {
           className="d-flex flex-column align-items-left justify-content-between"
         >
           <div className="h4 mb-0 d-flex align-items-start justify-content-between">
-            <span>Subscription Status</span>&nbsp; &nbsp;
-            <Badge variant={hasActiveLicenseOrLicenseRequest ? 'success' : 'danger'}>{hasActiveLicenseOrLicenseRequest ? 'Active' : 'Not Active'}</Badge>
+            <span>
+              <FormattedMessage
+                id="enterprise.dashboard.pathways.progress.page.subscription.status.label"
+                defaultMessage="Subscription Status"
+                description="Subscription status label displayed on the pathway progress page."
+              />
+            </span>&nbsp; &nbsp;
+            <Badge variant={hasActiveLicenseOrLicenseRequest ? 'success' : 'danger'}>
+              {hasActiveLicenseOrLicenseRequest
+                ? (
+                  <FormattedMessage
+                    id="enterprise.dashboard.pathways.progress.page.active.subscription.badge.label"
+                    defaultMessage="Active"
+                    description="Active subscription label for subscription card on the pathway progress page"
+                  />
+                )
+                : (
+                  <FormattedMessage
+                    id="enterprise.dashboard.pathways.progress.page.not.active.subscription.badge.label"
+                    defaultMessage="Not Active"
+                    description="Not active subscription label for subscription card on the pathway progress page"
+                  />
+                )}
+            </Badge>
           </div>
           {
             hasActiveLicenseOrLicenseRequest && expirationDate && (
-              <div className="subscription-expiry">Available until {' '}
+              <div className="subscription-expiry">
+                <FormattedMessage
+                  id="enterprise.dashboard.pathways.progress.page.subscription.card.available.until.label"
+                  defaultMessage="Available until"
+                  description="Available until label indicating the availability period for a subscription card on the pathway progress page"
+                />
+                {' '}
```

Comment:
How about below?
```javascript
<FormattedMessage
  id="enterprise.dashboard.pathways.progress.page.subscription.card.available.until.label"
  defaultMessage="Available until <b>{someFutureDate}</b>"
  description="Available until label indicating the availability period for a subscription card on the pathway progress page",
  values={{
    someFutureDate: dayjs(expirationDate).format('MMMM Do, YYYY'),
    b: chunks => <b>{chunks}</b>
  }}
/>
```

Reply:
@muhammad-ammar changed the approach to following, since I was getting ESLint error on this suggested approach.

```
<FormattedMessage
                  id="enterprise.dashboard.pathways.progress.page.subscription.card.available.until.date"
                  defaultMessage="Available until {expiryDate}"
                  description="Available until date indicating the availability period for a subscription card on the pathway progress page"
                  values={{
                    expiryDate: (
                      <span className="font-weight-bold">
                        {dayjs(expirationDate).format('MMMM Do, YYYY')}
                      </span>
                    ),
                  }}
                />
```

ESlint Error: Do not define components during render. React will see a new component type on every render and destroy the entire subtree’s DOM nodes and state.